pkg/cmd/cli/cmd/newapp.go

```diff
@@ -164,6 +167,7 @@ func NewCmdNewApplication(commandName string, f *clientcmd.Factory, out io.Write
 	cmd.Flags().StringSliceVarP(&config.TemplateParameters, "param", "p", config.TemplateParameters, "Specify a list of key value pairs (e.g., -p FOO=BAR,BAR=FOO) to set/override parameter values in the template.")
 	cmd.Flags().StringSliceVar(&config.Groups, "group", config.Groups, "Indicate components that should be grouped together as <comp1>+<comp2>.")
 	cmd.Flags().StringSliceVarP(&config.Environment, "env", "e", config.Environment, "Specify key-value pairs of environment variables to set into each container. This doesn't apply to objects created from a template, use parameters instead.")
+	cmd.Flags().StringVar(&config.EnvironmentFile, "env-file", "", "Specify a file containing key-value pairs of environment variables to set into each container. This doesn't apply to objects created from a template, use parameters instead.")
```

Comment:
@fabianofranz . Can I append it to cmd.MarkFlagFilename("file", "yaml", "yml", "json") so that it becomes something like cmd.MarkFlagFilename("file", "yaml", "yml", "json", "env-file")